src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,24 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+
+    private static final SessionFactory instance = initSessionFactory();
```

Comment:
Add a private default constructor to the `HibernateUtil` class in order to prevent creating `HibernateUtil` objects, as per the checklist.